src/Core/Ast/Parser/NikicPhpParser/NikicPhpParser.php

```diff
@@ -35,12 +35,14 @@ class NikicPhpParser implements ParserInterface
 
     /**
      * @param ReferenceExtractorInterface[] $extractors
+     * @param array<mixed> $config
      */
     public function __construct(
         private readonly Parser $parser,
         private readonly AstFileReferenceCacheInterface $cache,
         private readonly TypeResolver $typeResolver,
-        private readonly iterable $extractors
+        private readonly iterable $extractors,
```

Comment:
Can we keep the order of parameter consistent with `FileReferenceVisitor`?

Reply:
done